compiler/packages/babel-plugin-react-compiler/src/HIR/ObjectShape.ts

```diff
@@ -478,6 +521,90 @@ addObject(BUILTIN_SHAPES, BuiltInMixedReadonlyId, [
       noAlias: true,
     }),
   ],
+  [
```

Comment:
pasted these to `BuiltInMixedReadonlyId`, which is our read-only JSON-like object kind.

Happy to remove these and refactor later if we prefer to not have duplicates

Reply:
this is great!